src/app/clusters/level-control/level-control.cpp

```diff
@@ -182,6 +216,7 @@ void emberAfLevelControlClusterServerTickCallback(EndpointId endpoint)
     EmberAfLevelControlState * state = getState(endpoint);
     EmberAfStatus status;
     app::DataModel::Nullable<uint8_t> currentLevel;
+    const auto callbackDuration = chip::System::SystemClock().GetMonotonicTimestamp();
```

Comment:
Shouldn't that be called `callbackStartTimestamp` or something?

Reply:
Done